osi_common.proto

```diff
@@ -154,6 +154,9 @@ message BaseStationary
 
     /// The relative orientation of the landmark w.r.t its parent frame.
     optional Orientation3d orientation = 3;
+    
+    /// The two dimensional (flat) contour of the object. 
```

Comment:
@AndreasVIRES the reference point is already identical for all vehicles in the simulation. It never is rear axis, but always the middle of the bounding box.

You are right about the changing bounding box. however it practically has no impact on the sensors. Therefore treating it in a static manner is sufficient :)